schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+	id INT PRIMARY KEY,
```

Comment:
[optional] - i suggest you make the id column not null and autoincremented by using `    id int NOT NULL IDENTITY(1,1)
`

Reply:
Thank you for the suggestion